packages/core/src/__tests__/embrace.test.ts

```diff
@@ -84,7 +80,7 @@ jest.mock("../EmbraceManagerModule", () => ({
     logHandledError: (
       message: string,
       stackTrace: string,
-      properties?: Properties,
+      properties?: LogProperties,
```

Comment:
not sure if it matters but technically `properties: LogProperties` would match the real method's signature

Reply:
ah, yeap. let fix it